### ISSUE TITLE
Unable to install autocomplete

### ISSUE THREAD

**Issue by daaanny90** [Ephigenia/mite-cli#101]
## Context

- OS: MacOs Catalina 10.15.7
- Terminal: Bash and zsh
- mite-cli version: 1.5.0

## Description

I'm trying to install autocomplete, with the command `mite autocomplete install`.

### Expected Behavior

I should be able to choose the right shell type and go on with the installation.

### Current Behavior

The choice is shown, but I cannot choose the shell type, because the installation script stops before I can do anything.
```
Dannys-MacBook-Pro:~ danny$ mite autocomplete install
? Which Shell do you use ? (Use arrow keys)
❯ bash 
  zsh 
  fish Usage: mite-autocomplete [options] [command]

Options:
  -V, --version  output the version number
  -h, --help     output usage information

Commands:
  install        Installs auto-completion for most of the mite subcommands,
                 options and arguments. Compatible with bash, fish and zsh. No
                 support for windows systems. Also note that it will only work
                 when mite is installed globally.
  uninstall      removes the auto-completion for mite

Dannys-MacBook-Pro:~ danny$
```

## Steps to Reproduce

Just run `mite autocomplete install`, from bash or szh is the same.

**Comment by Ephigenia** [comment]
I will have to check this issue. I have an almost similar setup as yours (OS X, zsh, bash) and I was successful in setting up the auto completion. There might be a difference somewhere else.

Your post looks like you globally installed mite-cli. Is there something like nvm or other involved. 

Can you please post the output of `which mite-cli`

**Comment by Ephigenia** [comment]
Thanks for bringing this up and fillin out all the survey!

**Comment by Ephigenia** [comment]
Confirmed. I also tried this in a fresh installed docker container and the prompt doesn’t wait until the shell is chosen. I’ll see what I can find out.

**Comment by Ephigenia** [comment]
Should be fixed in [1.6.0](https://www.npmjs.com/package/mite-cli)

**Comment by Ephigenia** [comment]
@daaanny90 does it work for you know?
